### ISSUE TITLE
Stripe Terminal Android SDK version 3.0.0

### ISSUE THREAD

**Issue by christhwaites** [Square-Six/Xamarin.Stripe.Terminal.Android#5]
Hi There,

Are there any plans to update the bindings for the 3.0.0 version of the SDK? 

Version 2.s is in maintenance mode, and we'd like to upgrade to the latest major release.

No worries if not, but we just need to know if we should commission our own development of the bindings.

Thanks,
    Chris

**Comment by chris-oswald** [comment]
Yes, I plan to work on the bindings in the next few months. I'm working right now on moving my app to MAUI. Once I finish that, the plan is to tackle the bindings. I'm shooting for Nov or Dec.

**Comment by paulallington** [comment]
I don't suppose you've managed to do this at all?  I'm just starting a project and hitting issues including the package, which may be just a version problem

**Comment by chris-oswald** [comment]
@paulallington I haven't, there were a few things that trumped me working on this over the holidays. I actually just upgraded everything to .Net 8 and successfully moved my app to MAUI, so this is something I'll probably pick up this month. It looks like they changed a lot, so I'll have to see how much effort it will be to adopt everything. The Java bindings and lib versions are always fun to work through, but I did write a script to help me unwind the poms.

**Comment by paulallington** [comment]
@chris-oswald Ah nice!  That sounds like a fun project to upgrade everything!  If I can do anything to help support/collaborate in some way, I'd be happy to.  The Java bindings/libs I can never work out, but beyond that then I'd be happy to do something!  If not I shall sit tight this month

**Comment by chris-oswald** [comment]
@paulallington @christhwaites I got the Android 3.2.1 bindings to compile this weekend. I'm still working thru the references because it's crashing in some instances. Once I get that figured out I'll post the updated bindings. BTW, I'm only doing this for .Net 8 and Maui, but it should get anyone most of the way if they want it for Xamarin.

**Comment by paulallington** [comment]
@chris-oswald Amazing well done!

**Comment by ggbtvs** [comment]
@chris-oswald Please keep us posted as to your progress.  I too have to create Xamarin bindings, so any help moving forward would be appreciated.

**Comment by ggbtvs** [comment]
@chris-oswald  Are you trying to also get Tap To Pay working on Android?

**Comment by svaldetero-envoc** [comment]
> @chris-oswald Are you trying to also get Tap To Pay working on Android?

I have bindings for net8 maui with TTP support. I would have used this library but it wasn't updated when I needed it months ago. Here's the github: https://github.com/Envoc/envoc-stripe-binding/tree/main/src/StripeTerminal/Envoc.Stripe.Terminal.Android
And the nuget: https://www.nuget.org/packages/Envoc.Stripe.Terminal.Android

**Comment by chris-oswald** [comment]
@svaldetero Wow, this is great! I wish I would have looked around before starting to update the bindings. I appreciate all the work you've done Seth and will probably adopt your lib if you think you'll keep it maintained.

I'll be honest I only do this when I have time, and that's not often these days.

**Comment by svaldetero-envoc** [comment]
@chris-oswald I'll be honest, I'll keep it maintained as long as my client needs Stripe integration. Since I just made < 6 months ago and did V3+, I'll probably maintain it for a bit. 

Right now it has support for Tap To Pay (local readers), apps on device (handoff reader on the S700), bluetooth, and wifi readers.

**Comment by svaldetero-envoc** [comment]
@ggbtvs I slacked off on the documentation on my bindings, but if you have questions, feel free to ask over there as to not clutter this repo.
